torchaudio/__init__.py

```diff
@@ -34,6 +35,26 @@
     pass
 
 
+@_mod_utils.deprecated(
+    "Please remove the function call. Resource initialization is automatically handled")
```

Comment:
nit: "Please remove the function call to ``initialize_sox``. Resource initialization is now automatically handled."